README.md

```diff
@@ -1,177 +1,164 @@
+Now you are set to launch the instance. To do so, click on Networking and select Routes and then click on the URL which is under the location to launch the instance. 
 
 ![](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/images/route.png)
+<!--EM: Is this the right image?-->
 
 ### Register the instance of FPPredict Plus
 
-We can login using the default credentials as per the [configuration file](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/credentials.txt), accept the end user license agreement and will be directed to the next page per below. Click on `Download` and share the file with the Findability Sciences support team. The support team will send the license file (with 30 days validity) and needs to be uploaded using the `Upload File` option. We are all set to access the instance of FPPredict Plus. 
+Log in using the default credentials as per the [configuration file](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/credentials.txt) and accept the end user license agreement. You will be directed to the next page as the following image shows. Click **Download** and share the file with the Findability Sciences support team. 
+
+The support team will send the license file (with 30 days validity) and needs to be uploaded using the `Upload File` option. We are all set to access the instance of FPPredict Plus. 
 
 ![](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/images/sys-info.png)
 
-After uploading the license key, we can bring up the `FP Predict+` instance per below. The navigation pane on the `left` has four options namely Dashboard, Analytics, Dataset Management & License Information. We are good to go from here. 
+After uploading the license key, bring up the `FP Predict+` instance like below. The navigation pane on the left should have four options:  Dashboard, Analytics, Dataset Management & License Information.
 
 ![](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/images/fp-predict.png)
 
 `Note` :- We may have to repeat all the steps if there is a version upgrade as some of the components from setup does not support product upgrades.
 
 # Summary
 
-With this, we have come to the end of this tutorial to understand all the aspects of installation, configuration, setup and more to get started using the FPPredict Plus operator from Red Hat Marketplace on OpenShift cluster to solve usecases under AI. Red Hat Marketplace is a one stop platform for exploring multiple operators which will help to solve some of the complex usecases under different domains. 
+This tutorial showed you how to install, configure, set up and add storage to get started using the FPPredict Plus operator from Red Hat Marketplace on OpenShift cluster to solve usecases under AI. Red Hat Marketplace is a one stop platform for exploring multiple operators which will help to solve some of the complex usecases under different domains. 
 
```

Comment:
Overall, the suggested changes looks okay to me with my review comments. Please check for typos if any in the content.